test/integration/partials-test.js

```diff
@@ -0,0 +1,50 @@
+const assert = require('assert');
+const path = require('path');
+const dist = path.resolve(__dirname, '..', '..', 'dist');
+
+describe('Partials test', function() {
+  it('Partials include RSS urls', async function() {
```

Comment:
```suggestion
  it('includes RSS urls', function() {
```

Suggest using this style of test message throughout.

Also, do these need to be `async` functions? I don't think they use `await`.